doc/Rules.md

```diff
@@ -446,7 +446,7 @@ Note: [Markdown Extra](https://en.wikipedia.org/wiki/Markdown_Extra)-style footn
 For (example)[^1]
 ```
 
-Rationale: Reversed links are not rendered as usable links.
+Rationale: Reversed links are not rendered as useful links.
```

Comment:
Usable is the correct term here. "Useful" is a subjective assessment, whereas "usable" actually indicates that the target object is functioning.